torch/testing/_internal/common_utils.py

```diff
@@ -3134,6 +3134,15 @@ def _run_custom(self, result=None):
         # Is the class strict and compiling?
         strict_default = False
         should_reset_dynamo = False
+
+        # We disable size_asserts for test_ops since some tests fail
```

Comment:
Can we xfail instead ?

Reply:
A bit concern with xfail is, the tests would not capture other issues since all issues are hidden by the size/stride assertion failures.

But it does not matter if we want to fixnall those stride problems before landing this PR. cc @jansel